docs/sources/flow/reference/components/otelcol.processor.transform.md

```diff
@@ -0,0 +1,575 @@
+
+{{% admonition type="warning" %}}
+`otelcol.processor.transform` allows you to modify all aspects of your telemetry. Some specific risks are given below, 
+but this is not an exhaustive list. It is important to understand your data before using this processor.  
+
+- [Unsound Transformations][]: Transformations between metric data types are not defined in the [metrics data model][]. 
+To use these functions, you must understand the incoming data and know that it can be meaningfully converted 
+to a new metric data type or can be used to create new metrics.
+  - Although OTTL allows you to use the `set` function with `metric.data_type`, 
+    its implementation in the transform processor is a [no-op][].
+    To modify a data type, you must use a specific function such as `convert_gauge_to_sum`.
+- [Identity Conflict][]: Transformation of metrics can potentially affect a metric's identity,
+  leading to an Identity Crisis. Be especially cautious when transforming a metric name and when reducing or changing 
+  existing attributes. Adding new attributes is safe.
+- [Orphaned Telemetry][]: The processor allows you to modify `span_id`, `trace_id`, and `parent_span_id` for traces 
+  and `span_id`, and `trace_id` logs.  Modifying these fields could lead to orphaned spans or logs.
+
+[Unsound Transformations]: https://github.com/open-telemetry/opentelemetry-collector/blob/v0.85.0/docs/standard-warnings.md#unsound-transformations
+[Identity Conflict]: https://github.com/open-telemetry/opentelemetry-collector/blob/v0.85.0/docs/standard-warnings.md#identity-conflict
+[Orphaned Telemetry]: https://github.com/open-telemetry/opentelemetry-collector/blob/v0.85.0/docs/standard-warnings.md#orphaned-telemetry
+[no-op]: https://en.wikipedia.org/wiki/NOP_(code)
+[metrics data model]: https://github.com/open-telemetry/opentelemetry-specification/blob/main//specification/metrics/data-model.md
+{{% /admonition %}}
+
+## Usage
+
+```river
+otelcol.processor.transform "LABEL" {
+  output {
+    metrics = [...]
+    logs    = [...]
+    traces  = [...]
+  }
+}
+```
+
+## Arguments
+
+`otelcol.processor.transform` supports the following arguments:
+
+Name | Type | Description | Default | Required
+---- | ---- | ----------- | ------- | --------
+`error_mode` | `string` | How to react to errors if they occur while processing a statement. | `"propagate"` | no
+
+The supported values for `error_mode` are:
+* `ignore`: Ignore errors returned by statements and continue on to the next statement. This is the recommended mode.
+* `propagate`: Return the error up the pipeline. This will result in the payload being dropped from the Agent.
+
+## Blocks
+
+The following blocks are supported inside the definition of
+`otelcol.processor.transform`:
+
+Hierarchy | Block | Description | Required
+--------- | ----- | ----------- | --------
+output | [output][] | Configures where to send received telemetry data. | yes
```

Comment:
you may want to reorder this table so it matches the order of the sections below.